docker/sane-pdf-reports/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Last modified: Wed, 23 Mar 2022 12:00:00 +0200
+# Last modified: Tue, 28 Jun 2022 10:41:00 +0200
 FROM demisto/python3-deb:3.9.6.22912
```

Comment:
```suggestion
FROM demisto/python3-deb:3.10.4.31559
```

Reply:
then you need to update the Pipfile to use :
python_version = "3.10"

Reply:
the main Pipfile ? 
<img width="880" alt="image" src="https://user-images.githubusercontent.com/61918543/177696902-12ce6797-6d45-421b-819f-ac3d19d8189e.png">
cause i don't have one in the 'sane-pdf-reports' folder 
<img width="365" alt="image" src="https://user-images.githubusercontent.com/61918543/177696963-650cbc5e-2bcf-4929-a63d-58d5a89b09e6.png">

Reply:
Aha I see I thought you were doing the sane-doc-reports
in that case you are good but still I need to figure out why the build is not running. (I'm checking)

Reply:
> Aha I see I thought you were doing the sane-doc-reports in that case you are good but still I need to figure out why the build is not running. (I'm checking)

thanks !